lib/LANraragi/Plugin/Login/Fakku.pm

```diff
@@ -0,0 +1,60 @@
+package LANraragi::Plugin::Login::Fakku;
```

Comment:
Please add this package to the list of tested modules in https://github.com/Difegue/LANraragi/blob/dev/tests/modules.t